osu.Game/Overlays/OverlayTabControl.cs

```diff
@@ -40,12 +56,20 @@ protected OverlayTabControl()
             AddInternal(bar = new Box
             {
                 RelativeSizeAxes = Axes.X,
+                Margin = new MarginPadding { Top = -1 },
```

Comment:
Not sure what this is fixing but this is a hack. Probably not worth the one pixel.

Reply:
Without this, the coloured bar is displayed below the background box:
![image](https://user-images.githubusercontent.com/27722894/75719963-ff587c80-5cd5-11ea-9587-95e4c5955d51.png)

while on web it's inside it:
![image](https://user-images.githubusercontent.com/27722894/75720073-21ea9580-5cd6-11ea-8c90-350343a21ea4.png)

Reply:
This is probably the proper way to get rid of this:
```csharp
            AddInternal(bar = new Box
            {
                RelativeSizeAxes = Axes.X,
                Height = 2,
                Anchor = Anchor.BottomLeft,
                Origin = Anchor.BottomLeft
            });
```

FWIW, same thing probably goes for the expanding bar below:
```csharp
                    Bar = new ExpandingBar
                    {
                        Anchor = Anchor.BottomCentre,
                        Origin = Anchor.Centre,
                        ExpandedSize = 7.5f,
                        CollapsedSize = 0,
                    },
```

Reply:
The bar fix you suggested does work, but the `ExtendedBar` is still 1px too low using these values:

![image](https://user-images.githubusercontent.com/27722894/75802882-c1169800-5d7d-11ea-8479-08784acf2be6.png)

Reply:
I suppose we can bring back the -1 but I'd make the 2 height into a `public const EXPANDED_TAB_HEIGHT` and reference that instead by doing `-EXPANDED_TAB_HEIGHT / 2` so it's obvious what it does.

Reply:
Except the line we're seeing is using `Height = 1` as seen here:

https://github.com/ppy/osu/blob/6b284fb3e74fbfce50c96e1096563360b95ba9cd/osu.Game/Overlays/TabControlOverlayHeader.cs#L76-L85

I'll change this to be more readable (only the `ProfileSectionTabControl` uses `Height = 2` because having it in the base class is a bit misleading.

Reply:
Yeah I got confused because I looked at `OverlayTabControl` instead of `OverlayHeaderTabControl` by mistake. Should be half regardless.

Reply:
Half of 1 doesn't really help us here though.

Reply:
Does it not? Since sizes are floats I thought it might.

Reply:
We need a `-1` margin, `-0.5` looks just ever-so-slightly off at higher UI scales:

![image](https://user-images.githubusercontent.com/27722894/75807297-684afd80-5d85-11ea-994f-a3d5a92b0de7.png) ![image](https://user-images.githubusercontent.com/27722894/75807329-7731b000-5d85-11ea-88ea-ce292b4c879f.png)
(-0.5 on the left)

Reply:
OK, sure.